spec/shr_foundation.txt

```diff
@@ -211,7 +216,141 @@ Description:	"A composition represents a set of Entries sharing common provenanc
+EntryElement:   ValueSet
+Based on:       DomainResource
+Description:    "A set of codes drawn from one or more code systems"
+0..1 		    Url
+0..1            VersionString
+0..1            NameAsText
+0..1            Title
+1..1            Status from http://hl7.org/fhir/ValueSet/publication-status
+0..1            IsExperimental
+0..*            SubjectType
+0..1            LastUpdated
+0..1            PublisherName
+0..*            ContactDetail
+0..1            DescriptionMarkdown
+0..*            UseContext
+0..*            Jurisdiction
+0..1            IsImmutable
+0..1            PurposeMarkdown
+0..1            Copyright
+0..1            IsExtensible
+0..1            ContentLogicalDefinition
+0..1            ValueSetExpansion
+
+    Element:    IsImmutable
+    Description: "Indicates whether or not any change to the content logical definition may occur"  Value:      boolean
+
+    Element:    IsExtensible
+    Description: "Whether this is intended to be used with an extensible binding"
+    Value:      boolean
+
+    Element:    ContentLogicalDefinition
+    Description:    "Definition of the content of the value set."
+    0..1        LockedDate
+    0..1        IncludeInactiveCodes
+    1..*        IncludeCodes
+    0..*        ExcludeCodes
+
+        Element:    LockedDate
+        Description:    "If a locked date is defined, then the Content Logical Definition must be evaluated using the current version as of the locked date for referenced code system(s) and value set instances where ValueSet.compose.include.version is not defined."
+        Value:      date
+
+        Element:    IncludeInactiveCodes
+        Description:    "Whether inactive codes - codes that are not approved for current use - are in the value set. If inactive = true, inactive codes are to be included in the expansion, if inactive = false, the inactive codes will not be included in the expansion. If absent, the behavior is determined by the implementation, or by the applicable ExpansionProfile (but generally, inactive codes would be expected to be included)."
+        Value:      boolean
+
+        Element:    IncludeCodes
+        Description:    "Include one or more codes from a code system or other value set(s)."
+        Value:      CodeSet
+
+        Element:    ExcludeCodes
+        Description: "Exclude one or more codes from the value set based on code system filters and/or other value sets."
+        Value:      CodeSet
+
+        Element:    CodeSet
+        Description:    "A description of a set of codes, to be included or excluded from a value set."
+        0..1	CodeSystem
+        0..1 	CodeSystemVersion
+        0..*    CodeSetConcept
+        0..*    CodeSetFilter
+        0..*    ValueSetUri
+
+            Element:    CodeSetConcept
+            Description:    "Specifies a concept to be included or excluded."
+            0..1	CodeValue
+            0..1	DisplayText
+            0..*    Designation
+
+                Element:    Designation
```

Comment:
Cool!  Support for aliases, keywords and other.  This will help a lot with keyword/semantic tag searches.